src/main/kotlin/no/nav/klage/domain/KlageAnkeInput.kt

```diff
@@ -79,10 +84,17 @@ data class KlageAnkeInput(
             }
         } else null
     }
-
+    @JsonIgnore
     fun isKlage(): Boolean {
         return klageAnkeType == KlageAnkeType.KLAGE
     }
+
+    val deprecatedFields = listOf(saksnummer)
+
+    fun containsDeprecatedFields(): Boolean {
+        deprecatedFields.forEach { if (it != null) return true }
+        return false
+    }
```

Comment:
Tenker dette kan bli lettere å lese: `fun containsDeprecatedFields(): Boolean = deprecatedFields.any { it != null }`

Reply:
Enig, har endret denne.